rest_service.go

```diff
@@ -91,6 +91,27 @@ func (s *Service) App(ctx context.Context) *gimlet.APIApp {
 	return app
 }
 
+func (s *Service) ToggleCachePruning(v bool) {
+	s.cacheMutex.Lock()
+	defer s.cacheMutex.Unlock()
+
+	s.cacheOpts.Disabled = true
```

Comment:
Should be set to `v` instead of true? Also I feel like it should be `SetCachePruning` to be consistent with the other options

Reply:
done! good catch!